### PR TITLE
Docs sidebar

### PR DESCRIPTION
**Pull request by cdedreuille**
In this PR we are improving the global layout of the docs including the sidebar.
- New sidebar design
- Fixed header + new background colour
- Remove toolbar
- Made the sidebar content all open

CleanShot 2023-11-13 at 18 19 09@2x

### CODE REVIEW

File: src/components/layout/DocsLayout.tsx [hunk 1]
```diff
@@ -1,214 +1,136 @@
...
 
-  @media (min-width: ${breakpoint * 1.333}px) {
-    margin-top: 0;
+  ${minMd} {
+    width: 260px;
+    margin: 0;
+    padding-bottom: 0;
+    padding-right: 20px;
+    margin-right: 20px;
+    height: calc(100vh - 112px);
```
Comment (kylegach) [inline src/components/layout/DocsLayout.tsx:58]: Does this work with and without the Eyebrow being rendered?

Comment (cdedreuille) [reply src/components/layout/DocsLayout.tsx:58]: No it does not. But how often do you remove the eyebrow?

File: src/components/layout/PageLayout.stories.js [hunk 2]
```diff
@@ -33,13 +34,23 @@ Base.parameters = {
...
   </PurePageLayout>
 );
 
+export const DocsLayoutLong = () => (
+  <PurePageLayout
+    dxData={dxData}
+    pageContext={{ ...docsPageContext, layout: 'docs' }}
+    location={{ pathname: '/docs/' }}
+  >
+    <div style={{ height: 2000, border: '1px solid #ccc' }}>children</div>
```
Comment (kylegach) [inline src/components/layout/PageLayout.stories.js:50]: Classic. 🙂

Comment (cdedreuille) [reply src/components/layout/PageLayout.stories.js:50]: Yes I wanted to test with long content

File: src/components/layout/sidebar/Sidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+        </TopNav>
+      </nav>
+      <VersionSelector version={version} versions={versions} slug={slug} />
+      <Accordion.Root type="multiple" asChild>
+        <AccordionRoot>
+          {/* eslint-disable react/no-array-index-key */}
+          {docsTocWithLinkWrappers.map((lvl1, lvl1Index) => (
+            <Fragment key={lvl1Index}>
+              <NavItem level={1}>
+                {lvl1.type === 'heading' ? <a href={lvl1.pathSegment}>{lvl1.title}</a> : lvl1.title}
```
Comment (kylegach) [inline src/components/layout/sidebar/Sidebar.tsx:245]: Without this, the current top-level items that aren't headings (FAQ and Migrate to 7.0) do not render as links.

```suggestion
                { ['link', 'heading'].includes(lvl1.type) ? <a href={lvl1.pathSegment}>{lvl1.title}</a> : lvl1.title}
```

File: src/components/layout/DocsLayout.tsx (hunk 1 above)
Comment (kylegach) [reply src/components/layout/DocsLayout.tsx:58]: Fair point. We can optimize it later.

**Review by kylegach — APPROVED** [review]
Love it! Nice work.

**Pull request merged**